pages/index.jsx

```diff
@@ -65,12 +64,24 @@ function Home() {
     );
   });
 
-  const eventCards = EVENT_DATA.map((event) => (
+  const [eventData, setEventData] = useState([]);
+  const [loadingEvents, setLoadingEvents] = useState(true);
+
+  useEffect(() => {
+    if (loadingEvents) {
+      setLoadingEvents(false);
+      fetch('https://events.uf-ace.com/api/events/')
+        .then((response) => response.json())
+        .then(({ content }) => setEventData(content));
+    }
+  });
+
+  const eventCards = eventData.map((event) => (
```

Comment:
Login concerns? Or are we leaving this API open?

Reply:
This event data is public, that is by design